recipes/toolchain/activate.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+if [ "$(uname)" == "Darwin" ]
+then
+    # for Mac OSX
+    export CC=clang
+    export CXX=clang++
+    export MACOSX_VERSION_MIN="10.7"
+    export MACOSX_DEPLOYMENT_TARGET="${MACOSX_VERSION_MIN}"
+    export CXXFLAGS="${CXXFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+    export CXXFLAGS="${CXXFLAGS} -stdlib=libc++"
+    export LDFLAGS="${LDFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+    export LDFLAGS="${LDFLAGS} -stdlib=libc++ -lc++"
+    export LINKFLAGS="${LDFLAGS}"
+elif [ "$(uname)" == "Linux" ]
+then
+    # for Linux
+    export CC=gcc
+    export CXX=g++
+    export CXXFLAGS="${CXXFLAGS}"
+    export LDFLAGS="${LDFLAGS}"
+    export LINKFLAGS="${LDFLAGS}"
+else
+    echo "This system is unsupported by the toolchain."
+    exit 1
+fi
+
+export CFLAGS="${CFLAGS} -m${ARCH}"
+export CXXFLAGS="${CXXFLAGS} -m${ARCH}"
```

Comment:
Added these to help ensure the right target architecture is used. Currently we only support 64-bit, but this would be useful if we do use it for 32-bit or if others wish to use this package outside conda-forge for 32-bit builds.